applications/pusher/src/modules/pm_apple.erl

```diff
@@ -98,3 +110,8 @@ maybe_load_apns(App, ETS, CertBin) ->
             lager:error("Error loading apns ~p", [Reason]),
             'undefined'
     end.
+
+-spec apns_topic(kz_json:object()) -> binary().
+apns_topic(JObj) ->
+    TokenApp = kz_json:get_ne_binary_value(<<"Token-App">>, JObj),
+    re:replace(TokenApp, <<"\\.(?:dev|prod)$">>, <<>>, [{'return', 'binary'}]).
```

Comment:
@danielfinke the `.dev|.prod` replacement at the end of the TokenApp is restrictive. the `topic` can also be part of the app configuration the same way you did in `kz_endpoint` to get the extra headers